asmcli/commands/add-to-mesh.sh

```diff
@@ -89,7 +96,7 @@ add_all_to_mesh() {
         --gke-uri="${GKE_CLUSTER_URI}" \
         --enable-workload-identity
     else
-      { read -r -d '' MSG; fatal "${MSG}"; } <<EOF || true
+      { read -r -d '' MSG; warn "${MSG}"; } <<EOF || true
```

Comment:
this is tentative because the heredoc thing with while loop is coming back. If we are skipping all the clusters (because clusters are already registered to the specified fleet), it will enter the loop nonetheless. Last time we solve this by using for loop, but I am not sure how to read into multiple variables with for loop.

Reply:
Used while loop with `< <(do_stuff)` structure to avoid blank lines.